lib/sandbox/pmapi.js

```diff
@@ -205,6 +205,15 @@ function Postman (execution, onRequest, onSkipRequest, onAssertion, cookieStore,
          */
         response: execution.response,
 
+        /**
+         * Only available in on message sent and received scripts. pm.message contains information about
```

Comment:
Lets remove this part - Only available in on message sent and received scripts - as sandbox is currently unaware of such events.